scripts/e2e-test.sh

```diff
@@ -47,34 +47,32 @@ fi
 eval $(minikube docker-env)
 GOOS=linux make e2e-test-images
 
-helm template deploy/helm/eunomia-operator/ \
-  --set eunomia.operator.deployment.enabled= \
-  --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl apply -f -
-
-operator-sdk test local ./test/e2e --namespace "$TEST_NAMESPACE" --up-local --no-setup --verbose --go-test-flags "-tags e2e -timeout 20m"
-
-helm template deploy/helm/eunomia-operator/ \
-  --set eunomia.operator.deployment.enabled= \
-  --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
-
+# Eunomia setup
 helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.image.tag=dev \
```

Comment:
So ... I think this means we are actually running the container image from quay.io instead of the container image built from the local branch.

Reply:
Not in this case - in build-images.sh [freshly built images are always being tagged `dev`](https://github.com/KohlsTechnology/eunomia/blob/bab5f717b69c6599183ad4d9f7ac833859f14e74/scripts/build-images.sh#L43-L44).
I just wanted to be explicit here - when somebody reads this test script, he'll associate the `dev` tag with the development eunomia image (which he might've just modified).
Besides, the image can't be pulled from quay because of the `pullPolicy` set to `Never` one line below.

Reply:
Ok ... sounds good.